src/common/schedulers/slurm_commands.py

```diff
@@ -63,11 +64,18 @@
     + "grep -oP '^(NodeName=\\S+)|(NodeAddr=\\S+)|(NodeHostName=\\S+)|(?<!Next)(State=\\S+)|"
     + "(Partitions=\\S+)|(SlurmdStartTime=\\S+)|(LastBusyTime=\\S+)|(Reason=.*)|(######)'"
 )
+SCONTROL_SHOW_JOB_OUTPUT_AWK_PARSER = (
+    'awk \'BEGIN{{RS="\\n\\n" ; ORS="######\\n";}} {{print}}\' | '
+    + "grep -oP '^(JobId=\\S+)|(JobName=\\S+)|(UserId=\\S+)|(Account=\\S+)|(JobState=\\S+)|(RunTime=\\S+)|"
+    + "(StartTime=\\S+)|(EndTime=\\S+)|(Partition=\\S+)|(\\bNodeList=\\S+)|(\\bNodes=\\S+)|"
+    + "(CPU_IDs=\\S+)|(\\bGres=\\S+)|(######)'"
+)
```

Comment:
I know there are potential issues with having scontrol returning JSON, but have you tried it to see if it might work for this case? I think there is some effort in the scheduler team to get away from using text parsing on scontrol and switching to JOSN - it might be worth talking to them if you haven't done so already.